### PR TITLE
add some docs, use markdown

### PR DESCRIPTION
**Pull request by mattip**
- add markdown to the sphinx build
- add building, using documentation

### CODE REVIEW

File: docs/using_pydrofoil.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Using pydrofoil
+
+Pydrofoil accepts a compiled RISCV exectuable and will run it. Typically, the
+source code of the executable will test some RISCV feature and report
+success/failure.
```
Comment (mattip) [inline docs/using_pydrofoil.md:5]: Needs more work, either in this PR or maybe just merge as-is and extend in future enhancements?

**Comment by mattip** [comment]
Fixes #24

**Comment by cfbolz** [comment]
let's just merge it and refine. I'm trying to enable rtd previews of PRs first though.

**Comment by cfbolz** [comment]
hm, probably only works for newly created PRs, so I'm merging this one.

**Pull request merged**